CFC_DataCollector/recommender/traffic.py

```diff
@@ -0,0 +1,40 @@
+## A wrapper for the 511 traffic api
+import xml.etree.ElementTree as ET
+import urllib2, datetime
+
+def get_orig(orig_city):
+	url = "http://services.my511.org/traffic/getoriginlist.aspx?token=76495c63-cb01-4aac-af28-f717ffee11c8"
```

Comment:
The token shouldn't be hardcoded and checked in.
Other projects will need other tokens, and we don't want to share our token and leave it open to abuse.

@joshzarrabi, I am removing this file for now since it is not being used, please fix later.